BTC/mining-difficulty.md

```diff
@@ -0,0 +1,25 @@
+# P7 Mining Difficulty
```

Comment:
As good notes, key points should be highlighted for clarity.
Please involve more list with key words highlighted in this note.

Reply:
Please help me review again, thanks.